TreasureHunt/Assets/2.Scripts/Hunter/FieldView/TreasureSetupController.cs

```diff
@@ -4,64 +4,68 @@
 using System.Globalization;
 using SimpleJSON;
 
-public class TreasureSetupController : MonoBehaviour {
-
+public class TreasureSetupController : MonoBehaviour 
+{
 	public static TreasureSetupController instance = null;
+	public static string userGameTreasureData = "";
 
 	void Awake()
 	{
-		if (instance == null)
-
+		if (instance == null) 
+		{
 			instance = this;
-
-		else if (instance != this)
-
-			Destroy(gameObject);
-
-		DontDestroyOnLoad (gameObject);
-
-	}
-
-	void Start(){
-		if (!gameObject.GetComponent<NetworkManager> ().enabled) {
-			GetGameTreasure ("gg");		
+		} 
+		else if (instance != this) 
+		{
+			Destroy (gameObject);
```

Comment:
Is this the style for calling functions in C#?

The examples I see in [the style guide](https://msdn.microsoft.com/en-us/library/ff926074.aspx) see to suggest `Destroy(gameObject)` as the suggested way.

Here and elsewhere in your code.